packages/opentelemetry-semantic-conventions/src/experimental_attributes.ts

```diff
@@ -0,0 +1,2267 @@
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HOST_NAME = 'host.name';
+
+/**
+ * Type of host. For Cloud, this must be the machine type.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HOST_TYPE = 'host.type';
+
+/**
+ * State of the HTTP connection in the HTTP connection pool.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HTTP_CONNECTION_STATE = 'http.connection.state';
+
+/**
+ * The size of the request payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HTTP_REQUEST_BODY_SIZE = 'http.request.body.size';
+
+/**
+ * The total size of the request in bytes. This should be the total number of bytes sent over the wire, including the request line (HTTP/1.1), framing (HTTP/2 and HTTP/3), headers, and request body if any.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HTTP_REQUEST_SIZE = 'http.request.size';
+
+/**
+ * The size of the response payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HTTP_RESPONSE_BODY_SIZE = 'http.response.body.size';
+
+/**
+ * The total size of the response in bytes. This should be the total number of bytes sent over the wire, including the status line (HTTP/1.1), framing (HTTP/2 and HTTP/3), headers, and response body and trailers if any.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_HTTP_RESPONSE_SIZE = 'http.response.size';
+
+/**
+ * The name of the cluster.
+ *
+ * @experimental This attribute is experimental and is subject to change in minor releases of `@opentelemetry/semantic-conventions`.
+ */
+export const ATTR_K_8_S_CLUSTER_NAME = 'k8s.cluster.name';
```

Comment:
This is a bug in the code generator. We have to wait for the next version release.

Reply:
I was going to suggest manually changing this for now to help us move forward, but I see it has already been fixed in https://github.com/open-telemetry/weaver/pull/270, and the [release PR ](https://github.com/open-telemetry/weaver/pull/285)is open already as well, so hopefully we'll be able to update this soon 🤞

Reply:
Fixed